priv/templates/alert.eex

```diff
@@ -0,0 +1,102 @@
+defmodule <%= @module_name %>Web.Components.Alert do
```

Comment:
Could we just copy the original component and do replacement on the module name? It will help to reduce code duplication, and we don't have to do double work to maintain both files

Reply:
Fine. I think it's better to do a simple replacement of the module name. I'll remove all the EEx stuff and make the `salad.add` task work without templates.